consumer/consumer.go

```diff
@@ -51,6 +54,7 @@ func NewListener(addr string, opts Options) (Listener, error) {
 		Listener: lis,
 		opts:     opts,
 		msgPool:  mPool,
+		m:        newConsumerMetrics(opts.InstrumentOptions().MetricsScope()),
```

Comment:
nit: `newListenerMetrics`?

Reply:
it's actually consumer metrics, just created at listener level so that it can be shared by all consumers